views/apartments/post.erb

```diff
@@ -0,0 +1,22 @@
+<h2><a href="/">View All Apartments</a></h2>
+
+<h2>New Apartment</h2>
+
+<form action="/apartments" method="post" >
+  <label>Address:</label>
+  <input name="apartment[address]">
+
+  <label>Monthly Rent:</label>
+  <input name="apartment[monthly_rent]">
+
+  <label>Sq. Feet:</label>
+  <input name="apartment[sqft]">
+
+  <label>Beds:</label>
+  <input name="apartment[num_beds]">
+
+  <label>Baths:</label>
+  <input name="apartment[num_baths]">
+
+  <input type="submit" value="Create">
+</form>
```

Comment:
i don't think you use this template at all. Delete it! Looks the same as `new.erb` form wise